styles/style.css

```diff
@@ -1,5 +1,254 @@
-* {
+*{
   margin: 0;
   padding: 0;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+  background-color: rgb(244, 244, 245);
+}
+.header {
+  margin-left: 12%;
+  margin-right: 12%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 20px 0px 20px 0px;
+  background-color: rgb(244, 244, 245);
+  border-bottom: 1px solid rgb(244, 244, 245);
+}
+.header .left-section, 
+.header .logo, 
+.header .right-section {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.header .left-section {
+  justify-content: flex-start;
+}
+
```

Comment:
left-section class'ı için `width: 200px` değeri verebilirsin ki arama kutusu açılınca tasarım kaymasın